### DISCUSSION TITLE
Datatable filter, perpage, search, etc only working on the first row of the datatable unless the view is reloaded.

### DISCUSSION THREAD

**Discussion by ianyeahhhh** [rappasoft/laravel-livewire-tables#1721] [Q&A]
I am using this with laravel x hybrix template but will work only if I reloaded the page. Is there any fix you can suggest?
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/102967235/163b1cae-b018-4a27-94a8-03578c162ab7)

![image](https://github.com/rappasoft/laravel-livewire-tables/assets/102967235/99824a63-4aad-4893-90c6-b91ec2cbe982)

**Comment by lrljoe** [comment]
What does your full layout blade look like?  Are you using injection for livewire?  What about the tables package?
